src/sentry/lint/engine.py

```diff
@@ -115,6 +111,7 @@ def get_python_files(file_list=None):
 # parseable is a no-op
 def py_lint(file_list, parseable=False):
     from flake8.api.legacy import get_style_guide
+    register_pycodestyle_checks()
```

Comment:
Is this really the fix for newer flake8? it just needs to be registered after flake8 is imported?

Reply:
No, it isn't. I just moved the lazy loading to `py_lint`, since that is where it is needed. This PR isn't addressing the flake8 issue, just something I wandered off doing since it was relevant and pretty easy.